config/rails_best_practices.yml

```diff
@@ -34,7 +34,9 @@ RemoveUnusedMethodsInModelsCheck: { except_methods: [
     'ReviewTurnaround#build'
   ] }
 ReplaceComplexCreationWithFactoryMethodCheck: { attribute_assignment_count: 2 }
-ReplaceInstanceVariableWithLocalVariableCheck: { }
+ReplaceInstanceVariableWithLocalVariableCheck: { ignored_files: [
+  'views/development_metrics/_sidebar.erb'
```

Comment:
I'd rather fix it instead of ignoring it. Partials shouldn't use instance variables, you can pass them via locales. https://andycroll.com/ruby/dont-use-instance-variables-in-partials/

Reply:
Well 😅, I tried doing it, but it forces me to use at least one `@`. I think it's because of the multiple contexts, but maybe I'm missing something. Take a look at this flow:
`departments.erb -> _development_metrics/development_metrics_sidebar -> _sidebar.erb`.

Reply:
what forced you to do so? In `departments.erb` it's fine to use it since it's the root view, but then you should pass that as a `local` to the partials https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables

Reply:
@sebastiancaraballo this comment wasn't resolved